.gitignore

```diff
@@ -136,4 +137,6 @@ website/www/yarn-error.log
 **/*.tfvars
 
 # Ignore Katas auto-generated files
-**/*-remote-info.yaml
\ No newline at end of file
+**/*-remote-info.yaml
+**/*.backup
```

Comment:
I suggest separating this into a separate commit (same PR is fine) so that it creates a clearer history and will not be rolled back if there is any issue.

Reply:
Ok